### PR TITLE
substarte++

### PR DESCRIPTION
**Pull request by fbielejec**

### CODE REVIEW

File: bin/runtime/src/lib.rs [hunk 1]
```diff
@@ -118,6 +119,9 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 1,
+    /// Version of the state implementation used by this runtime.
+    /// Use of an incorrect version is consensus breaking.    
+    state_version: 1,
```
Comment (Marcin-Radecki) [inline bin/runtime/src/lib.rs:124]: We can spawn a task for it, but I guess update procedure to testnet/mainnet needs to be careful thought.

File: bin/runtime/src/lib.rs [hunk 2]
```diff
@@ -470,6 +494,8 @@ pub const TREASURY_BURN: u32 = 0;
 pub const TREASURY_PROPOSAL_BOND: u32 = 0;
 // The proposer should deposit max{`TREASURY_PROPOSAL_BOND`% of the proposal value, $10}.
 pub const TREASURY_MINIMUM_BOND: Balance = 1000 * CENTS;
+// NOTE: I pulled this value from the air
```
Comment (h4nsu) [inline bin/runtime/src/lib.rs:497]: Maybe add here that this value needs to be understood and possibly adjusted before landing on {test,main}net? Probably with some caps and exclamation marks ;)

File: bin/runtime/src/lib.rs [hunk 3]
```diff
@@ -343,7 +349,25 @@ impl pallet_scheduler::Config for Runtime {
...
+
+// impl pallet_preimage::Config for Runtime {
+//     type WeightInfo = pallet_preimage::weights::SubstrateWeight<Runtime>;
+//     type Event = Event;
+//     type Currency = Balances;
+//     type ManagerOrigin = EnsureRoot<AccountId>;
+//     type MaxSize = PreimageMaxSize;
+//     type BaseDeposit = PreimageBaseDeposit;
+//     type ByteDeposit = PreimageByteDeposit;
+// }
```
Comment (h4nsu) [inline bin/runtime/src/lib.rs:370]: Kick out?

**Review by h4nsu — APPROVED** [review]
LGTM. Comments I left are about comments, feel free to ignore and resolve

File: bin/runtime/src/lib.rs (hunk 2 above)
Comment (fbielejec) [reply bin/runtime/src/lib.rs:483]: done

**Pull request merged**